routes/api/v1/sessions.js

```diff
@@ -0,0 +1,31 @@
+var express = require("express");
+var router = express.Router();
+var User = require('../../../models').User;
+var bcrypt = require('bcrypt');
+
+
+router.post("/", function(req, res, next) {
+  User.findOne({where: {email: req.body.email} })
+    .then(user => {
+      if (user == null) {
+        res.status(500).send("This user does not exist.");
+      } else {
+        bcrypt.compare(req.body.password, user.password, function(err, result) {
+          if (result == true) {
+            res.setHeader("Content-Type", "application/json");
+            res.status(201).send({ api_key: user.api_key });
+          } else {
+            res.setHeader("Content-Type", "application/json");
+            res.status(500).send("Incorrect password");
```

Comment:
This is a minor thing, but the HTTP status code for an incorrect password or nonexistent user should be a `401`. A `500` indicates that something went wrong in the server software, and is typically a catchall for server errors that the developer doesn't anticipate. A `401` indicates that the request format was valid, but that the requestor was not authorized to complete the requested action.

Reply:
Thank you @joequnicy I will definitely update that before merging.